### PR TITLE
keep bash history of php service by storing it on the host system

### PR DESCRIPTION
**Pull request by niklasnatter**
builds upon #25

### CODE REVIEW

File: config/php-fpm/Dockerfile [hunk 1]
```diff
@@ -51,6 +50,12 @@ COPY composer/config.json /var/www/.composer/config.json
 # Copy xdebug and php config.
 COPY conf.d/* /usr/local/etc/php/conf.d/
 
+# Fix permissions and umask
+RUN chown -R www-data /var/www
```
Comment (niklasnatter) [inline config/php-fpm/Dockerfile]: i am not sure if these permissions are best practice. but something like this is needed in order that the www-data user is allowed to write the `var/www/.bash_history` file

Comment (wachterjohannes) [reply config/php-fpm/Dockerfile]: why not set the persmission only on the file?

**Review by wachterjohannes — CHANGES_REQUESTED** [review]
could you also rebase the PR?

**Comment by niklasnatter** [comment]
i was able to remove the permission related commands alltogether. therefore, this pr should be ready for merging now

**Pull request merged**